app/livestream/page.tsx

```diff
@@ -0,0 +1,93 @@
+          <div className="bg-xport-orange-primary basis-[45%] rounded-r-full flex">
+            <div className="bg-xport-gray-alternate basis-[70%] px-5 py-3 rounded-r-full flex justify-between items-center">
+              <div className="flex items-center gap-3">
+                <div className="h-10 w-10 bg-xport-black-light animate-pulse rounded-full"></div>
+                <h3 className="font-medium">Evos Esport</h3>
+              </div>
+              <p className="font-bold text-xport-orange-light">2.73</p>
+            </div>
+            <div className="flex basis-[30%] justify-center items-center font-medium">
+              Bet Now
+            </div>
+          </div>
+          <div className="basis-[10%] flex justify-center items-center italic">
+            vs
+          </div>
+          <div className="bg-xport-orange-primary basis-[45%] rounded-l-full flex">
+            <div className="flex basis-[30%] justify-center items-center font-medium">
+              Bet Now
+            </div>
+            <div className="bg-xport-gray-alternate basis-[70%] px-5 py-3 rounded-l-full flex justify-between items-center">
+              <p className="font-bold text-xport-orange-light">1.40</p>
+              <div className="flex items-center gap-3">
+                <h3 className="font-medium">RRQ Esport</h3>
+                <div className="h-10 w-10 bg-xport-black-light animate-pulse rounded-full"></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="flex flex-col gap-3">
+          <div className="font-semibold flex items-center">
+            <h3>
+              <span className="text-xport-orange-primary">
+                Mobile Legends : Bang Bang
+              </span>{' '}
+              Ongoing Livestream{' '}
+            </h3>
+            <span>
+              <ArrowRightIcon className="w-3 h-3 ml-2" />
+            </span>
+          </div>
+          <div className="grid grid-cols-3 gap-3">
+            <div className="w-full aspect-video bg-xport-black-light animate-pulse rounded"></div>
+            <div className="w-full aspect-video bg-xport-black-light animate-pulse rounded"></div>
+            <div className="w-full aspect-video bg-xport-black-light animate-pulse rounded"></div>
+          </div>
+        </div>
+      </section>
+      <section className="md:basis-[30%] lg:basis-[20%] flex flex-col justify-between bg-xport-black-primary h-[50vh] md:h-[90vh] border-l border-b border-xport-gray-primary">
+        <div className="py-5 px-3">
+          <h2 className="font-semibold text-center mb-5">Bet Slip</h2>
+          <BetSlipCard />
+        </div>
+        <div className="bg-xport-orange-primary text-xs flex items-center">
+          <div className="bg-xport-gray-alternate w-2/3 flex flex-col gap2 pl-3 pr-4 py-5 rounded-r-full">
+            <div className="flex justify-between">
+              <span>Stake</span>
```

Comment:
the stake and the payout should be dynamic, regading the changed that dropdown h
![Uploading Screen Shot 2023-09-20 at 18.03.14.png…]()
as been made

Reply:
<img width="289" alt="Screen Shot 2023-09-20 at 18 03 26" src="https://github.com/galihpermana29/x-sports/assets/54941568/1285ed4e-ef4f-4098-8951-fed15e2be325">
also on that screen, the active bet should be related on whats button the 
![Uploading Screen Shot 2023-09-20 at 18.04.05.png…]()
user click in this ui